backend/controllers/cardsController.js

```diff
@@ -0,0 +1,25 @@
+const User = require('../UserModel');
+const cardsController = {};
+
+cardsController.getCards = async (req, res, next) => {
+  //retrieve cards collection data from DB
+  const cards = await User.findOne({ username });
```

Comment:
I would add some error handling

Reply:
I swapped card model/route/controller tasks with James, here's what we have for getCards

`async getCards(req, res, next) {
    // res.send('Getting cards...');
    try {
      const cards = await Card.find();
      console.log(Card);
      res.status(200).json(cards);
    } catch (e) {
      return next({
        log: 'Error getting cards in cardController',
        status: 500,
        message: { err: e.message },
      });
    }
  },`